frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/import-providers-tab/providers/vm-import-password.tsx

```diff
@@ -105,10 +104,13 @@ const stateToProps = (state, { wizardReduxID, provider }) => {
     (!validationField || !isValidationCritical);
 
   if (provider === VMImportProvider.OVIRT && hasAllPrerequisiteValuesFiled) {
-    hasAllPrerequisiteValuesFiled = !!iGetOvirtFieldValue(
+    hasAllPrerequisiteValuesFiled = !iGetProviderFieldAttribute(
```

Comment:
why iGetProviderField? We don't need to use this function since ovirt does not have a certificate. We should also take into account empty certificate - either by checking the value or by returning TrivialError in the validation (you can use validateEmptyValue)

Reply:
now using both @suomiy